packages/wct-mocha/src/index.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * @license
+ * Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
```

Comment:
2018? or is this actually a moved file that git isn't picking up?

Reply:
You know, you're right.  Technically this one was a new file.  Updating date.

Reply:
Updating all the (c) to 2018